ansible_base/lib/serializers/common.py

```diff
@@ -11,16 +11,13 @@
 logger = logging.getLogger('ansible_base.lib.serializers.common')
 
 
-COMMON_FIELDS = ['id', 'url', 'created', 'created_by', 'modified', 'modified_by', 'related', 'summary_fields']
-
-
-class CommonModelSerializer(ValidationSerializerMixin, serializers.ModelSerializer):
+class AbstractCommonModelSerializer(ValidationSerializerMixin, serializers.ModelSerializer):
```

Comment:
So I think we can do this, but I think I'd like to make this a separate patch. Because it's going to need tests to test the whole hierarchy and so on. It's going to add quite a bit to an already big PR. Filed #246 for this.